src/main/java/com/gotocompany/firehose/utils/KafkaConnectorType.java

```diff
@@ -0,0 +1,14 @@
+package com.gotocompany.firehose.utils;
+
+import lombok.Getter;
+
+@Getter
+public enum KafkaConnectorType {
+    DLQ("DLQ_KAFKA");
```

Comment:
1. lets keep the value and documentation in sync i.e. here it is DLQ_KAFKA and in documentation it is `DLQ_KAFKA_`
2. Add test cases to the getPattern method, the pattern should match for DLQ_KAFKA_ABC and not for DLQ_KAFKAABC or ABCDLQ_KAFKA_, and all other negative case scenarios.

Reply:
agreed, missed on this part